frb/galaxies/offsets.py

```diff
@@ -25,14 +27,23 @@ def angular_offset(frb, galaxy, nsigma=5., nsamp=2000):
     # Error ellipse
     sig_a = frb.eellipse['a']  # arcsec
     sig_b = frb.eellipse['b']  # arcsec
+    pa_ee = frb.eellipse['theta'] # deg
 
     # If systematic exists, add in quadrature
     if 'a_sys' in frb.eellipse.keys():
         sig_a = np.sqrt(frb.eellipse['a_sys']**2 + sig_a**2)
         sig_b = np.sqrt(frb.eellipse['b_sys']**2 + sig_b**2)
 
+    # Add in Galaxy error : THIS TREATMENT IS APPROXIMATE
+    if gal_sig is not None:
+        # Project
+        sig2_gal_a = gal_sig[1]**2 * np.cos(pa_ee)**2 + gal_sig[0]**2 * np.sin(pa_ee)**2
```

Comment:
sqrt around the gal + FRB uncertainty squares?

Reply:
nope